lib/types.ts

```
@@ -55,6 +55,14 @@ export type WorkflowSettings = {
      * Exposes the fetch method to call extenal APIs to the workflow
      */
     "kinde.fetch"?: {};
+    /**
+     * Exposes the fetch method to call signed extenal APIs to the workflow
+     */
+    "kinde.secureFetch"?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)